pandas/tseries/offsets.py

```diff
@@ -1576,13 +1584,15 @@ class QuarterOffset(DateOffset):
     #       point
 
     def __init__(self, n=1, normalize=False, startingMonth=None):
-        self.n = n
+        self.n = self._validate_n(n)
         self.normalize = normalize
         if startingMonth is None:
             startingMonth = self._default_startingMonth
         self.startingMonth = startingMonth
 
         self.kwds = {'startingMonth': startingMonth}
+        self._offset = timedelta(1)
+        self._use_relativedelta = False
```

Comment:
should ``_use_relative_delta`` be a class variable?

Reply:
and ``_offset`` ?

Reply:
`_use_relativedelta` is defined at the class level, defaults to `False`.  So setting again here is unnecessary.

The bigger issue with `_relativedelta`&`_offset` is that they are only actually relevant for `DateOffset` (i.e. not subclasses).  It took me a while to figure this out (and in fact I screwed up several PRs back by changing `BusinessFoo.offset` --> `BusinessFoo._offset` "for consistency" without realizing the business version is used differently).  My current thought is to separate out the `relativedelta`-using parts of `DateOffset` into a subclass that _isnt_ inherited by everything else to avoid this confusion.